webapp/src/pages/Landing/Landing.tsx

```diff
@@ -11,8 +13,38 @@ import { Color } from '../../shared/types';
 import './Landing.css';
 
 export default function Landing() {
+  const btn_icon = (
+    <Icon
+      variant={IconVariant.LOGOUT}
+      color={Color.LIGHT}
+      size={IconSize.SMALL}
+    ></Icon>
+  );
+  const btn_action = (): void => alert('This is an alert');
+  const btn_link = (): void => {
+    window.location.href = 'http://google.com';
+  };
   return (
     <section className="Landing">
+      <Button variant={ButtonVariant.SUBMIT} onClick={btn_action}>
+        ALERT
+      </Button>
+      <Button variant={ButtonVariant.WARNING} icon={btn_icon} disabled>
+        LOGOUT BUTTON
+      </Button>
+      <Button
+        variant={ButtonVariant.SUBMIT}
+        icon={
+          <Icon
+            variant={IconVariant.ARROW_FORWARD}
```

Comment:
Yo creo que el color de esta flecha no da la idea de como queremos usarla. Llevo un rato intentando tunearlo y lo unico que me sale es simplemente `color={Color.LIGHT}`, pero... hay alguna manera de hacer que el color del texto y del icono vayan a la par? Porque tengo la impresión de que eso es lo que queremos...

Reply:
With `color={Color.LIGHT}`  in the icon , that is equal to ` 'color-light'`,you should get exactly the same color as in the text, because the text color is `color: var(--color-light);`. I only changed it to dark to try something different in the Landing page examples.